ci/jenkins/jobs/macros.yaml

```diff
@@ -328,10 +328,10 @@
           export KUBECONFIG=$WORKSPACE/jenkins/out/kubeconfig
 
           kubectl apply -f build/yamls/antrea.yml
-          kubectl rollout restart deployment/coredns -n kube-system
-          kubectl rollout status deployment/coredns -n kube-system
-          kubectl rollout status deployment.apps/antrea-controller -n kube-system
-          kubectl rollout status daemonset/antrea-agent -n kube-system
+          timeout 5m kubectl rollout restart deployment/coredns -n kube-system
+          timeout 5m kubectl rollout status deployment/coredns -n kube-system
```

Comment:
`kubectl rollout status` supports a `--timeout` parameter, but I don't if there is any advantage in using it over what you currently have

Reply:
Sure, good to know there's a "--timeout" param. It's better use to it. Done.